.github/workflows/seahorn-coverage.yml

```diff
@@ -0,0 +1,29 @@
+# workflow to run coverage tests
+name: Coverage
+# Controls when the action will run
+on:
+  schedule:
+    - cron: 0 0 * * *  # run every day at UTC 00:00
```

Comment:
I believe cron actions only work on `master` branch. So this probably should be submitted to `master` even though it will run `dev10`. This is what we do for nightly builds right now